src/dbvt/mod.rs

```diff
@@ -291,11 +291,7 @@ enum Node<B> {
 impl<T> DynamicBoundingVolumeTree<T>
 where
     T: TreeValue,
-    T::Bound: Clone
-        + Debug
-        + Contains<T::Bound>
-        + Union<T::Bound, Output = T::Bound>
-        + SurfaceArea,
+    T::Bound: Clone + Debug + Contains<T::Bound> + Union<T::Bound, Output = T::Bound> + SurfaceArea,
```

Comment:
this is also a regression

Reply:
Same here as above, no config switch for it, and for this one I agree, looks better with the old version.